qiskit_experiments/curve_analysis/base_curve_analysis.py

```diff
@@ -162,6 +162,10 @@ def _default_options(cls) -> Options:
                 instance that defines the `self.__call__` method.
             normalization (bool) : Set ``True`` to normalize y values within range [-1, 1].
                 Default to ``False``.
+            average_method (str): Method of the y value averaging when the same x values
+                appear multiple times. One of "sample", "iwv", "shots_weighted".
```

Comment:
What is "iwv"? Can you spell it out "iwv"?

Reply:
It stands for Inverse-weighted variance. This is the option name and I cannot change. But link will appear in the documentation.